### ISSUE TITLE
[BUG] Error when importing config files with non existing name

### ISSUE THREAD

**Issue by sigit2306** [CoinAlpha/hummingbot#2047]
**Describe the bug**
Version: 0.29.0, Windows Binary

On a fresh start-up of the instance, when trying to import a saved strategy, accidentally pressed a button on the keyboard thus (conf_pure_mm_3.yml ---> conf_pure_mm_34.yml that does not exist) the import fails and Hummingbot becomes unresponsive.
The lower left terminal still accepts and shows user inputs, but upper left pane doesn't show information regards to user input.
`status`   `import`   `exit`  command doesn't have an effect.

**Steps To Reproduce**
As an example, I have a saved strategy file named conf_pure_mm_3.yml

Here are the steps to reproduce the issue (see attachments in the section below):
1. Fresh start-up 
2. Start the bot using this command: import conf_pure_mm_34.yml (accidentally pressed button number '4')

```
responds in log window: 11:38:59 - async_utils - Unhandled error in background task: [Errno 2] No such file or directory: 'conf/conf_pure_mm_34.yml' (See log file for stack trace dump)
```

**Screenshots**
![HB_0290_01](https://user-images.githubusercontent.com/12313129/87504724-709b3980-c691-11ea-8341-372c8d7cc92b.png)
![HB_0290_02](https://user-images.githubusercontent.com/12313129/87504819-a6d8b900-c691-11ea-9e43-4614fac71d45.png)
![HB_0290_03](https://user-images.githubusercontent.com/12313129/87504838-adffc700-c691-11ea-9aaf-6545d609eebe.png)

**Release version**
Windows Binary, Version 0.29.0

**Optional: your discord username:**
sigit#7187

**Log file**
[logs_hummingbot.log](https://github.com/CoinAlpha/hummingbot/files/4922987/logs_hummingbot.log)

**Comment by Clint-G13** [comment]
Test issue and found out that it happens when importing using one-line command and also, the validator is not working at this state.
![1](https://user-images.githubusercontent.com/57189990/87505614-ddb3cc80-c69b-11ea-81af-1d8137a12c28.PNG)
Entering import first to display import question --> validator work
![2](https://user-images.githubusercontent.com/57189990/87505641-f1f7c980-c69b-11ea-9596-0e581a352aa9.PNG)

using one-line command --> error persist
![3](https://user-images.githubusercontent.com/57189990/87505664-fd4af500-c69b-11ea-8f72-a8d0b7cb9a16.PNG)

 version affected
0.29.0
0.30.0

**Comment by RC-13** [comment]
Fixed on version 0.31.0, file name validator has been added:
![image](https://user-images.githubusercontent.com/57189990/92669885-5609d780-f345-11ea-9d2a-486bebba07c4.png)

**Comment by dennisocana** [comment]
QA confirmed this is no longer happening in version 0.31
